File: Python/Condicional/Avanzado/Ejercicio_02.py
# Leer tres longitudes
lado1 = float(input("Ingrese la longitud del lado 1: "))
lado2 = float(input("Ingrese la longitud del lado 2: "))
lado3 = float(input("Ingrese la longitud del lado 3: "))

# Verificar si las longitudes forman un triángulo
if lado1 < lado2 + lado3 and lado2 < lado1 + lado3 and lado3 < lado1 + lado2:
    # Determinar el tipo de triángulo
    if lado1 == lado2 == lado3:
        print("Es un triángulo equilátero")
    elif lado1 == lado2 or lado1 == lado3 or lado2 == lado3:
        print("Es un triángulo isósceles")
    else:
        print("Es un triángulo escaleno")
else:
    print("No es un triángulo")

File: PHP/ESTRUCTURA_IF/Avanzado/Ejercicio_02.php
<?php
    $lado1 = 3;
    $lado2 = 4;
    $lado3 = 5;

    if ($lado1 < $lado2 + $lado3 && $lado2 < $lado1 + $lado3 && $lado3 < $lado1 + $lado2) {
        if ($lado1 == $lado2 && $lado2 == $lado3) {
            echo "Es un triángulo equilátero";
        } elseif ($lado1 == $lado2 || $lado1 == $lado3 || $lado2 == $lado3) {
            echo "Es un triángulo isósceles";
        } else {
            echo "Es un triángulo escaleno";
        }
    } else {
        echo "No es un triángulo";
    }
?>